test/Microsoft.ML.AutoML.Tests/TunerTests.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+using FluentAssertions;
+using Microsoft.ML.SearchSpace;
+using Microsoft.ML.TestFramework;
+using Xunit;
+using Xunit.Abstractions;
+using Microsoft.ML.AutoML.CodeGen;
+
+namespace Microsoft.ML.AutoML.Test
+{
+    public class TunerTests : BaseTestClass
```

Comment:
>TunerTests [](http://example.com/codeflow?start=17&length=10)

You are missing tests for the other tuners, the array math, etc. Those need to be added.

Reply:
The tests for cfo tuner is migrated from model builder repo and I will put more tests for other tuners and entire automl experiment soon in a separate PR.